pkg/telemetryquerylanguage/functions/tqlotel/func_split_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tqlotel
+
+import (
+	"testing"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/tql"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/tql/tqltest"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_split(t *testing.T) {
```

Comment:
Can you add a test case for empty string and empty delimiter?

Reply:
@TylerHelmuth Yes. 
* https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/e4acc8d1b4b21968748fff97d382edafb9a9548e/pkg/telemetryquerylanguage/functions/tqlotel/func_split_test.go#L43-L51 For empty string 
* https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/e4acc8d1b4b21968748fff97d382edafb9a9548e/pkg/telemetryquerylanguage/functions/tqlotel/func_split_test.go#L53-L61 For empty delimiter

Reply:
@linjunzhe sorry I meant a single test where both the string and the delimiter are empty string at the same time.

Reply:
@TylerHelmuth Sure. Added the test case.